src/main/webapp/WEB-INF/static/js/fieldbook-common.js

```diff
@@ -2278,33 +2278,24 @@ function closeModal(modalId) {
 
 function openGermplasmDetailsPopopWithGidAndDesig(gid, desig) {
 	'use strict';
-	$.ajax({
-		url: '/Fieldbook/ListTreeManager/germplasm/detail/url',
-		type: 'GET',
-		data: '',
-		cache: false,
-		success: function(html) {
-			var germplasmDetailsUrl = html;
-			showGermplasmDetailsPopUp(gid, desig, germplasmDetailsUrl);
-		}
-	});
+
+	// TODO: Refactor germplasm details popup to Angular
+	const germplasmDetailsURL = '/ibpworkbench/main/app/#/germplasm-details/' + gid + '?cropName=' + cropName
```

Comment:
Remove` url.germplasm.details` from fieldbook.properties (and related usage in FieldbookProperties class and Fieldbook-servlet.xml)

Reply:
This is done.